### PR TITLE
Database

### PR DESCRIPTION
**Pull request by SusanSu123**

### CODE REVIEW

File: views/item.handlebars [hunk 1]
```diff
@@ -1,26 +1,4 @@
...
-             <a href="#Delete">Delete</a>   
-            <a href="#item">add item</a>
-            <a href="#Update">Update</a>
-            <a class="active" href="#home">Home</a>
-        </div>
-
-    <main>
-
-        <div class="search container search">
+       <div class="search container search">
```
Comment (JWCoad) [inline views/item.handlebars:1]: Put these on separate lines

**Review by JWCoad — CHANGES_REQUESTED** [review]
index html has conflicts

**Review by SusanSu123 — COMMENTED** [review]
reviewed

**Review by JWCoad — APPROVED** [review]

**Review by JWCoad — APPROVED** [review]
removed outdated file

**Comment by SusanSu123** [comment]
try later

**Pull request closed without merging**